spec/views/hq/sponsors/edit_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'hq/sponsors/edit.html.haml', type: :view do
+  before :each do
+    assign :sponsor, build_stubbed(:sponsor)
+    assign :statuses, []
+    assign :sponsor_types, []
+    assign :organizations, []
+    assign :branches, []
+    assign :cities, []
+  end
```

Comment:
I added `[ ]` instead of ex: `Status.all` just to speed up things. I am checking for the presence of these lists in the `_form` partial anyway.